install.sh

```diff
@@ -39,8 +39,14 @@ __entry() {
 	_REINSTALL_RITA=false
 
 	# Optional Dependencies
-	_INSTALL_BRO=true
-	_INSTALL_MONGO=true
+	if [ "$_UBUNTU_VERSION" == "Release:	18.04" ]
+	then
+		_INSTALL_BRO=false
+		_INSTALL_MONGO=false
```

Comment:
There may be cases where Mongo is not installed.  For instance, I doubt that the server version of Ubuntu will have it pre-installed or the user may have uninstalled it at some point.

So instead of skipping the Mongo install, could you instead
1. Add the checks (if necessary) [here](https://github.com/activecm/rita/blob/2839b50ae482b133ee2347d98f49ff4b79d18788/install.sh#L348-L353) that will properly set the `_MONGO_INSTALLED` variable on 18.04.
2. Add any special casing [here](https://github.com/activecm/rita/blob/2839b50ae482b133ee2347d98f49ff4b79d18788/install.sh#L224-L235) to handle a proper install routine on 18.04.